Bottom: 4628b82df6e1459b891cfb045e283234ea98dc48
Top:    15a05840b6b8833d698b206be3dc3b1ef280db39
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-08 20:52:23 +0000

Try debugging draw speed



---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 2cc3a27..b41a356 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1560,7 +1560,7 @@ Benchmark (int argc, char **argv, Coord x, Coord y)
     }
   while (end - start < 10);
 
-  printf (_("%g redraws per second\n"), i / 10.0);
+  printf (_("%g redraws per second\n"), (double)i / (double)(end-start));
 
   return 0;
 }
